hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/LoggingUtil.java

```diff
@@ -50,6 +50,12 @@ public static void logFine(ILogger logger, String template, Object arg1, Object
         }
     }
 
+    public static void logFine(ILogger logger, String template, Object... args) {
```

Comment:
We didn't have this variant because it creates a temporary `Object[]` for the 3rd argument. Maybe just add a variant with 4 arguments.